src/Textarea.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import cx from 'classnames';
+import autosize from 'autosize';
+
+/**
+ * @module Textarea
+ */
+class Textarea extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value || '',
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onChange(e) {
+		const { onChange } = this.props;
+		if (onChange) {
+			onChange(e);
+		}
+		this.setState({
```

Comment:
pretty close, i think mikes comment was slightly off.
you still want to set the state and then call onChange

https://github.com/meetup/meetup-web-components/pull/134/files#diff-f9ace01f309ab9526a85b595c571df9eR17